test/feature2mesh_unit_test.js

```diff
@@ -55,4 +55,19 @@ describe('Feature2Mesh', function () {
                 noHoleArea - holeArea,
                 meshWithHoleArea);
         }));
+
+    it('square polygon should have double the vertices for being extruded', () =>
+        parse().then((feature) => {
+            const noExtrude = Feature2Mesh.convert()(feature[0]);
+            const extrude = Feature2Mesh.convert({
+                extrude: 2,
+            })(feature[0]);
+
+            const indexExtrude = new Uint16Array([5, 4, 7, 7, 6, 5, 0, 4, 1, 1,
+                4, 5, 1, 5, 2, 2, 5, 6, 2, 6, 3, 3, 6, 7]);
+
+            assert.equal(noExtrude.geometry.attributes.position.count, 4);
+            assert.equal(extrude.geometry.attributes.position.count, 8);
+            assert.equal(extrude.geometry.index.array.every((e, i) => indexExtrude[i] === e), true);
```

Comment:
You can verify the number of triangles instead.

It's less likely to change: it should be 2 triangles per face of the quad/cube. So 2 triangles for the `noExtrude` case and 6*2 triangles for the `extrude`  one.

Reply:
This makes more sense than verifying vertices, indeed.